src/common/ceph_context.cc

```diff
@@ -750,6 +762,10 @@ CephContext::CephContext(uint32_t module_type_,
   _admin_socket->register_command("perf histogram dump name=logger,type=CephString,req=false name=counter,type=CephString,req=false", _admin_hook, "dump perf histogram values");
   _admin_socket->register_command("2", _admin_hook, "");
   _admin_socket->register_command("perf schema", _admin_hook, "dump perfcounters schema");
+  _admin_socket->register_command("labeledperf dump name=logger,type=CephString,req=false name=counter,type=CephString,req=false", _admin_hook, "dump labeled perfcounters value");
+  _admin_socket->register_command("labeledperf schema", _admin_hook, "dump labeled perfcounters schema");
```

Comment:
do we really need separate commands? do the labels break backward compatibility with the existing commands?